src/saving_tools.jl

```diff
@@ -200,6 +199,7 @@ Dict{Symbol,Any} with 3 entries:
 function tag!(d::AbstractDict{K,T};
         gitpath = projectdir(), force = false, source = nothing,
         storepatch::Bool = readenv("DRWATSON_STOREPATCH", false),
+        mssg::Bool = false
```

Comment:
`mssg` is not a clear keyword. PLease name it something that conveys its purpose better just as `commit_message`

Reply:
Alright, thank you for your feedback. I already changed the mssg keyword to commit_message